### DISCUSSION TITLE
ASCII Art Creation

### DISCUSSION THREAD

**Discussion by jmuthalaly** [grtcdr/macchina#250] [Q&A]
Hello, I was hoping to find out how the ASCII art here was created?
https://github.com/Macchina-CLI/macchina/tree/main/contrib/ascii

I was hoping to create a similar colored design for Kubuntu.

**Comment by grtcdr** [comment]
I can't remember which tool @uttarayan21 used, maybe he can tell us :)

**Reply by uttarayan21** [reply]
I created them using [jp2a](https://github.com/cslarsen/jp2a)

**Comment by timhansinger** [comment]
In case you want to create it manually and also want to use your terminal colors to colorize I found this source helpful to get the colors right: [https://misc.flogisoft.com/bash/tip_colors_and_formatting#colors](https://misc.flogisoft.com/bash/tip_colors_and_formatting#colors)

I wanted to have the macOS Retro logo from Archey4 ([https://github.com/HorlogeSkynet/archey4/blob/master/archey/logos/darwin.py](https://github.com/HorlogeSkynet/archey4/blob/master/archey/logos/darwin.py)) and wrote this ASCII file:

```bash
[1m
[32m               ###
[32m             ####
[32m             ###
[32m    #######    #######
[33m  ######################
[33m #####################
[31m ####################
[31m ####################
[31m #####################
[35m  ######################
[35m   ####################
[34m     ################
[34m      ####     #####

 [37m ██[36m ██[35m ██[34m ██[33m ██[32m ██[31m ██
```

Which results in this logo:
<img width="218" alt="Screen Shot 2022-06-08 at 13 09 41" src="https://user-images.githubusercontent.com/7524165/172602332-0edf7007-788e-43a6-9d47-b609aa99b935.png">
